### PR TITLE
Tomee 2494 - Add translation to PT-BR to security-examples directory

### PR DESCRIPTION
**Pull request by alexferreiradev**
I am newbie in translation contribution. But I created one Jira issue TomEE-2494 and I will resolve with that PR. I put some comments below.

### CODE REVIEW

**Comment by Daniel-Dos** [comment]
It's very good for me.

thanks @alexferreiradev

**Comment by alexferreiradev** [comment]
Eu tentei explicar da melhor forma possível, mas como não tenho prática nesta área de segurança, gostaria que alguém validasse a parte inglês que adicionei, pricipalmente o teste 2 e 3.

**Comment by alexferreiradev** [comment]
O teste 4 não tinha arquivo. Adicionei um, mas com o status not-published, pois não olhei o código para verificar do que se trata o exemplo.

**Comment by alexferreiradev** [comment]
> It's very good for me.
> 
> thanks @alexferreiradev

But I had not finished yet. Now, I finish. Can you review again, please :grimacing:

File: examples/testing-security/README.adoc [hunk 1]
```diff
@@ -58,6 +59,8 @@ public class Movie {
 
 == Movies
 
+After, we created a DAO class. Look that we used the security anotations to set the need roles to use each DAO action. For example, adition of a movie is able to a user that has "Manager" or "Employee" `ROLE`, but to remove a movie the user need to has Manager `ROLE`.
+Criamos outra classe DAO para Filme. Observe que utilizamos as anotações para dizer as roles necessárias para utilização de cada ação disponível. Por exemplo, adicionar um filme é possível para um usuário que tenha a `ROLE` "Manager" ou "Employee", mas a remoção de um filme pode ser feita somente por quem tem a `ROLE` Manager.
```
Comment (Daniel-Dos) [inline examples/testing-security/README.adoc]: here this in portuguese .

Comment (alexferreiradev) [reply examples/testing-security/README.adoc]: I corrected that. Thanks for review.

**Comment by alexferreiradev** [comment]
> ei um, mas com o status not-published, pois não olhei o código para verificar do que se trata o exemplo.

Is these status corrected ? If no, I can remove these file from the PR.

**Comment by Daniel-Dos** [comment]
Hi @alexferreiradev ,

about the status, I think can  leave it as it is to moment .

you can change the title of PR with a small description of same ? 

ex  :  Tomee 2494 - something

thanks .

**Comment by alexferreiradev** [comment]
Done

**Comment by Daniel-Dos** [comment]
wonderful, thanks :  )

**Comment by cesarhernandezgt** [comment]
Hi @alexferreiradev 

Thank you for your contribution, in order to move forward this PR please open an email thread on the mailing list to make reference to both the Jira and the PR. 

In this way, we collaborate with the committers to move forward the contributions and follow the Apache Way "If it didn't happen on a mailing list, it didn't happen." [1] along with the recommended Apache TomEE Workflow [2].

[1] https://community.apache.org/newbiefaq.html#NewbieFAQ-IsthereaCodeofConductforApacheprojects?
[2] http://tomee.apache.org/community/contributing/workflow.html

**Comment by alexferreiradev** [comment]
@cesarhernandezgt are you said to me send a e-mail with the name of the issue and PR in the content ? Did I do that in corrected way ?, because I have been sent.

**Comment by cesarhernandezgt** [comment]
> @cesarhernandezgt are you said to me send a e-mail with the name of the issue and PR in the content ? Did I do that in corrected way ?, because I have been sent.

@alexferreiradev 
 yes, you did it right. I wrote a follow up in you email thread with the subject `Issue TOMEE-2494
`. At this point we need to wait for a committer to do the merge.

@Daniel-Dos Thank you for the Pr's reviews, you are now doing the next step as PR reviewer, congrats!

**Comment by Daniel-Dos** [comment]
Hi @cesarhernandezgt , 

your welcome

**Pull request merged**